src/hotspot/cpu/x86/vm_version_x86.hpp

```diff
@@ -677,6 +677,7 @@ class VM_Version : public Abstract_VM_Version {
   static uint32_t cpu_stepping()          { return _cpuid_info.cpu_stepping(); }
   static int  cpu_family()        { return _cpu;}
   static bool is_P6()             { return cpu_family() >= 6; }
+  static bool is_P6_or_later()    { return cpu_family() == 6 || cpu_family() == 18 || cpu_family() == 19; }
```

Comment:
``` suggestion
  static bool is_intel_server_family()    { return cpu_family() == 6 || cpu_family() == 18 || cpu_family() == 19; }
```
We already have is_P6(), which returns true for CPU family >=6, a minor name change suggestion.

Reply:
Please see the updated code with the naming change as suggested.